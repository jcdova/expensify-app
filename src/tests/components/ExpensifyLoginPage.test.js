import React from 'react';
import { shallow } from 'enzyme';
import { ExpensifyLoginPage } from '../../components/ExpensifyLoginPage';

test('should correctly render ExpensifyLoginPage', () => {
    const wrapper = shallow(<ExpensifyLoginPage />);
    expect(wrapper).toMatchSnapshot();
});

test('should call startLogin on button click', () => {
	const startLogin = jest.fn();
	const wrapper = shallow(<ExpensifyLoginPage startLogin={startLogin} />);
	wrapper.find('button').simulate('click');
	expect(startLogin).toHaveBeenCalled();
});