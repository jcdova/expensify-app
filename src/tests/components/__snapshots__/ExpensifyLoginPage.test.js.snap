// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpensifyLoginPage 1`] = `
<div>
  <button>
    Login
  </button>
</div>
`;
